### ISSUE TITLE
change meaning of implicit missings for covariances and cross loadings.

### ISSUE THREAD

**Issue by aaronpeikert** [formal-methods-mpi/Taxonomy.jl#122]
https://github.com/formal-methods-mpi/Taxonomy.jl/blob/edb1d5a8f2b4110a58fe27a985139db1905419a1/src/taxons/measurement.jl#L50C4-L53C36

**Comment by lkosanke** [comment]
@aaronpeikert 

Am I missing something or do we not just have to change the documentation for the four arguments to make sure that we apply an empty array vs. missing in the right way? The following works already:

julia> Measurement(n_variables = 3, factor_variance = missing, loadings = missing, error_covariances_within = Float64[],quest_scale = J(missing,0.5,"weird"))

Measurement
   n_variables: JudgementInt{Int64}
   loadings: JudgementVecNumber{Missing}
   factor_variance: JudgementNumber{Missing}
   error_variances: JudgementVecNumber{Missing}
   error_covariances_within: JudgementVecNumber{Vector{Float64}}
   error_covariances_between: JudgementVecNumber{Missing}
   crossloadings_incoming: JudgementVecNumber{Missing}
   crossloadings_outgoing: JudgementVecNumber{Missing}
   quest_scale: JudgementNumber{Missing}

**Comment by aaronpeikert** [comment]
You are absolutely right. This is only a documenting issue (not code changes nessesary!)
